ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -1582,6 +1617,183 @@ impl Inst {
         }
     }
 
+    /// Duplicate this [Inst] while applying function `f` to each operand.
+    pub(crate) fn dup_and_remap_locals<F>(
+        &self,
+        m: &mut Module,
+        f: &F,
+    ) -> Result<Self, CompilationError>
+    where
+        F: Fn(InstIdx, &Module) -> Operand,
+    {
+        let mapper = |x: &PackedOperand, m: &Module| match x.unpack(m) {
```

Comment:
We always make `m` the first argument (it's a useful convention IMHO).

Reply:
[1975eae](https://github.com/ykjit/yk/pull/1510/commits/1975eae011eb4e365fb6e417f3165a709b4cd89d)